mercury/static/mercury/js/simulator.js

```diff
@@ -114,6 +116,28 @@ $(function () {
         return curr_date + " " + curr_time;
     }
 
+    //This function returns next sensor value, also makes sure that the value is in between 0 and 100
+    function getNextValue(sensorValue, min, max){
```

Comment:
Does the `min` and `max` here show how far away the "next value" can be from the current sensory value, or does it provide the new minimum and maximum range that the next sensor number can be in?

Reply:
it's the range for random number, like if we provide min=-5 and max=5 the random number being generated will be in that range but if the sensor value is close to 0 then the getNextValue() function assigns min = 0 so that the next sensor value won't be less than zero